src/server.js

```diff
@@ -56,100 +56,82 @@ http.createServer(function (request, response) {
+            nextGoal = goal;
+    }
 
-                var campaign = findObject(json.included, json.data.relationships.campaign.data)
+    status.earnings = campaign.attributes.pledge_sum / 100;
+    status.patrons = campaign.attributes.patron_count;
 
-                var goals = campaign.relationships.goals.data;
-                var nextGoal = null;
+    if (nextGoal)
+        status.next_goal = nextGoal.attributes.amount_cents / 100;
 
-                for (var i = 0; i < goals.length; ++i) {
-                    var goal = findObject(json.included, goals[i])
-                    if (goal.attributes.completed_percentage < 100 && (nextGoal === null || nextGoal.attributes.amount_cents > goal.attributes.amount_cents))
-                        nextGoal = goal;
-                }
+    status.updated = new Date().toUTCString();
+}
 
-                status.earnings = campaign.attributes.pledge_sum / 100;
-                status.patrons = campaign.attributes.patron_count;
-                if (nextGoal)
-                    status.next_goal = nextGoal.attributes.amount_cents / 100;
-                status.updated = new Date().toUTCString();
-            } catch (err) {
-                console.log(err);
-            }
-        });
-    });
+async function refreshLiberapayStatus(account) {
+    const req = await fetch(process.env.LIBERAPAY_URL);
 
-    req.on("error", function(e) {
-        console.log("problem with request: " + e.message);
-    });
+    const res = await req.json();
 
-    req.end();
+    status["liberapayEarnings_" + account] = res.receiving.amount
+    status["liberapayPatrons_" + account] = res.npatrons
+    status["liberapayUpdated_" + account] = new Date().toUTCString();
 }
 
-function refreshLiberapayStatus(account) {
-    // https://liberapay.com/Tiled/public.json
-    var options = {
-        hostname: "liberapay.com",
-        path: "/" + account + "/public.json",
-        rejectUnauthorized: false,
-        agent: false,
-    };
-
-    var req = https.request(options, function(res) {
-        var body = ""
-
-        res.on("data", function(chunk) { body += chunk; });
-
-        res.on("end", function() {
-            try {
-                var json = JSON.parse(body);
-
-                status["liberapayEarnings_" + account] = json.receiving.amount
-                status["liberapayPatrons_" + account] = json.npatrons
-                status["liberapayUpdated_" + account] = new Date().toUTCString();
-            } catch (err) {
-                console.log(err);
-            }
-        });
-    });
+async function refreshGithubSponsors() {
 
-    req.on("error", function(e) {
-        console.log("problem with request: " + e.message);
+    const query = `
+    query {
+        user(login: "bjorn") {
+        ... on Sponsorable {
+                sponsors(first: 10) {
+                    totalCount
+                }
+            },
+            estimatedNextSponsorsPayoutInCents
```

Comment:
```suggestion
            monthlyEstimatedSponsorsIncomeInCents
```
We want that value, because `estimatedNextSponsorsPayoutInCents` is a pretty bad estimate, in that it resets to 0 every month and counts up until the payout. So it's a very bad estimate of the next payout. :-)